science/nco/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           compilers 1.0
 PortGroup           github 1.0
 
-github.setup        nco nco 4.9.1
+github.setup        nco nco 4.9.2
 revision            1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
Yes, of course. Done.